src/RL/AST.hs

```diff
@@ -2,14 +2,22 @@ module RL.AST where
 
 import RL.Values
 
-type Program label store = (VariableDecl, [Block label store])
+type Program label store = [Process label store]
 
+--TODO: remove when implementing support for PE with processes
 data VariableDecl = VariableDecl
   { input  :: [Name]
   , output :: [Name]
   , temp   :: [Name]
   } deriving (Eq, Show, Read)
 
+
+data Process label store = Process 
```

Comment:
Process -> Procedure (og alle relaterede steder)
Føler det er en god ide lige at få det rigtige navn på plads før der bliver bygget alt for meget.